conans/client/conf/__init__.py

```diff
@@ -68,14 +70,26 @@
         threads: [None, posix, win32] #  Windows MinGW
         exception: [None, dwarf2, sjlj, seh] # Windows MinGW
         cppstd: [None, 98, gnu98, 11, gnu11, 14, gnu14, 17, gnu17, 20, gnu20]
-    Visual Studio:
+        intel:
+            None:
+            compatible:
+                version: [None, "11", "12", "13", "14", "15", "16", "17", "18", "19"]
```

Comment:
The additional `None` value here is to be able to do things like `conan install . --profile visual -s compiler.intel=compatible` without specifying the intel compiler version

Reply:
is it possible to use an anchor to avoid repetition of intel versions 4 times?

Reply:
In this case, we have the additional value of None, so I don't know if using anchoring is possible. Anyway, I don't like the settings structure either, but this is just a proposal.